Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Read the Docs - Environment base
 FROM ubuntu:22.04
 LABEL mantainer="Read the Docs <support@readthedocs.com>"
-LABEL version="ubuntu-22.04-2023.03.09"
+LABEL version="ubuntu-22.04-2024.01.29"
```

Comment:
We will need to re-build `ubuntu-20.04` as well to install the Ruby plugin on `asdf`.

Reply:
Medium scary... hopefully it works.

Reply:
I opened https://github.com/readthedocs/readthedocs-docker-images/pull/201 for this.